services/docker-stack-as-ap-machshift.yml

```diff
@@ -0,0 +1,34 @@
+version: "3.7"
+services:
+  tb-ps:
+    image: dockerregistry.lnls-sirius.com.br/fac/fac-iocs:2020-02-26
+    command: bash -c 'source /etc/profile && /ioc-apps/as-ap-macshift.bash'
```

Comment:
typo here? /ioc-apps/as-ap-macshift.bash --> /ioc-apps/as-ap-mac**h**shift.bash

Reply:
yes, this typo was still lurking. thanks.